### PR TITLE
Update README for local development

### PR DESCRIPTION
**Pull request by JeremyTsaii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,43 +7,43 @@ Check out <https://hyperschedule.io>!
...
-      make build   Compile JavaScript for production
-      make dev     Start development server and automatically recompile JavaScript
-      make docker  Start shell or run command (e.g. make docker CMD="make dev")
-      make format  Auto-format JavaScript
-      make lint    Verify that all code is correctly formatted
-      make ci      Run tests that CI will run
-      make help    Show this message
+Other npm commands available:
+
+      npm run build
```
Comment (ingridt123) [inline README.md]: Might be helpful to include the descriptions of the commands that were removed.
Also, npm run shows the list of commands available, which would also be useful.

Comment (JeremyTsaii) [reply README.md]: Hi Ingrid, great suggestions. I will go ahead and add those soon.

File: README.md [hunk 2]
```diff
@@ -7,43 +7,45 @@ Check out <https://hyperschedule.io>!
...
-      make clean   Remove build artifacts
-      make hooks   Install Git hooks
-      make build   Compile JavaScript for production
-      make dev     Start development server and automatically recompile JavaScript
-      make docker  Start shell or run command (e.g. make docker CMD="make dev")
-      make format  Auto-format JavaScript
-      make lint    Verify that all code is correctly formatted
-      make ci      Run tests that CI will run
-      make help    Show this message
+Other npm commands available:
```
Comment (ingridt123) [inline README.md:38]: Thanks for the change! I think it looks like the list of commands aren't correctly formatted.

Screenshot 2020-11-05 at 11 46 27 AM

Comment (JeremyTsaii) [reply README.md:38]: Oops, fixed!

**Comment by kwshi** [comment]
Nice, thanks.

**Comment by kwshi** [comment]
Oh, actually, a comment on Markdown preferences--could you change the code block to use triple-backticks instead of indentation?

**Pull request merged**